src/components/localisation/routes.ts

```diff
@@ -16,6 +16,7 @@ export type LanguageOption = {
   languageName: string
   description: string
   code: string
+  locale: string
```

Comment:
```suggestion
  locale?: string
```
We probably want to make this optional and just have a default. That'll save us having to update existing snapshots.